scripts/mkcloud

```diff
@@ -1020,7 +1020,7 @@ Optional
         Examples:
             3 clusters: clusterconfig='data=2:network=3:services=5'
             2 clusters: clusterconfig='services+data=2:network=3'
-            1 cluster:  clusterconfig='data+network+services=2'
+            1 cluster:  clusterconfig='services+data+network=2'
```

Comment:
Or at least, the commit message justifying it should be changed.  I'm OK with it suggesting that the cluster is named `services`, but it shouldn't still work if it's called `data` or even `network`.